### PR TITLE
Reduce risk of sensitive information leak in rpc debug/error log

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?
In some rpc error or debug enabled, some credential would be printed, those kinds of information should be masked.
This change is to mask the credential before printing it to log.

### Why are the changes needed?
Without the fix, there is a potential credential leak issue.

### Does this PR introduce any user facing changes?

No any user facing.

### CODE REVIEW

**Comment by yyongycy** [comment]
@ggezer @jiacheliu3  please help take a look.

**Comment by yyongycy** [comment]
The message dumped after masked:
blockHeartbeat: request=bitField0_:3
workerId_:7292113489303911161
properties{
com.google.protobuf.MapField@129b5
}
removedBlockIds_:{
[]
}
metrics_:{
[instance: "Worker"
source: "alluxios-mbp"
name: "BlocksEvictionRate"
value: 0.0
metricType: METER
]
}
properties{
com.google.protobuf.MapField@aaab831d
}
memoizedIsInitialized:1
addedBlocks_:{
[]
}
memoizedIsInitialized:1

**Comment by yyongycy** [comment]
'alluxio-bot, check this please'

**Comment by yyongycy** [comment]
> alluxio-bot, check this please

alluxio-bot, check this please

**Comment by yyongycy** [comment]
pr was updated accordingly.

File: core/server/common/src/main/java/alluxio/RpcUtils.java [hunk 1]
```diff
@@ -36,6 +37,9 @@
 public final class RpcUtils {
   private RpcUtils() {} // prevent instantiation
 
+  public static final SensitiveConfigMask SENSITIVE_MASKER =
```
[the remark this answers is not in the document]
Comment (jiacheliu3) [reply core/server/common/src/main/java/alluxio/RpcUtils.java:40]: `SENSITIVE_MASKER` grammatically does not make sense. Please just fix it. It is not so hard.

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]

**Review by jiacheliu3 — APPROVED** [review]

File: core/server/common/src/main/java/alluxio/RpcUtils.java [hunk 2]
```diff
@@ -120,7 +126,10 @@ private RpcUtils() {} // prevent instantiation
         MetricsSystem.counter(getQualifiedFailureMetricName(methodName)).inc();
         if (!logger.isDebugEnabled()) {
           logger.warn("Exit (Error): {}: {}, Error={}", methodName,
-              String.format(description, args), e.toString());
+              String.format(description,
+                  SENSITIVE_CONFIG_MASKER == null
```
Comment (ggezer) [inline core/server/common/src/main/java/alluxio/RpcUtils.java:130]: Please move this macro into a method, e.g processArgs(), and use it instead.

**Review by ggezer — APPROVED** [review]
Thanks!

**Comment by jiacheliu3** [comment]
Tests all passed and comments resolved

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
@ggezer Which branches should this be cherry-picked to?

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.6 please

**Pull request merged**